bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -201,14 +202,13 @@ private func dummy() {
     _ = NSLocalizedString("Navigate Up", comment: "")
     _ = NSLocalizedString("Navigates to custom color picker", comment: "")
     _ = NSLocalizedString("Navigates to customize the gradient", comment: "")
-    _ = NSLocalizedString("Navigates to select %s", comment: "translators: %s: Select control button label e.g. \"Button width\"")
+    _ = NSLocalizedString("Navigates to select %s", comment: "translators: %s: Select control button label e.g. Small\ntranslators: %s: Select control button label e.g. \"Button width\"")
```

Comment:
- Modification
- [Code reference 1](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/font-size-picker/index.native.js#L86:L86)
- [Code reference 2](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/mobile/bottom-sheet-select-control/index.native.js#L63:L63)

This string is used in two places and a different comment is provided from each one, for this reason, the value of the comment property contains two lines.